### PR TITLE
UWP camera module for react native

### PR DESCRIPTION
**Pull request by elmouzak**
Adding separate RN module: react-native-windows-uwp-camera 

moved Example media capture library inside the module

### CODE REVIEW

File: windows-camera/react-native-windows-uwp-camera/react-native-windows-uwp-camera.podspec [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  s.summary      = package["description"]
+  s.description  = <<-DESC
+                  react-native-windows-uwp-camera
+                   DESC
+  s.homepage     = "https://github.com/github_account/react-native-windows-uwp-camera"
+  # brief license entry:
+  s.license      = "MIT"
+  # optional - use expanded license entry instead:
+  # s.license    = { :type => "MIT", :file => "LICENSE" }
+  s.authors      = { "Your Name" => "yourname@email.com" }
```
Comment (dipidoo) [inline windows-camera/react-native-windows-uwp-camera/react-native-windows-uwp-camera.podspec:19]: is this podspec for local consumption or for publishing?

File: windows-camera/react-native-windows-uwp-camera/uwpCamera/ExampleMediaCaptureUWP/ExampleMediaCaptureUWP.csproj [hunk 2]
```diff
@@ -1,233 +1,233 @@
...
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
+    <ProjectGuid>{FC9E9589-DAC4-4EC7-A5E3-1549CA13EB6C}</ProjectGuid>
+    <OutputType>AppContainerExe</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>ExampleMediaCaptureUWP</RootNamespace>
+    <AssemblyName>ExampleMediaCaptureUWP</AssemblyName>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.18362.0</TargetPlatformVersion>
```
Comment (dipidoo) [inline windows-camera/react-native-windows-uwp-camera/uwpCamera/ExampleMediaCaptureUWP/ExampleMediaCaptureUWP.csproj:25]: is this change intentional?

Comment (dipidoo) [reply windows-camera/react-native-windows-uwp-camera/uwpCamera/ExampleMediaCaptureUWP/ExampleMediaCaptureUWP.csproj:25]: actually changing the target is OK, at a glance I thought the min version was changed

File: windows-camera/react-native-windows-uwp-camera/react-native-windows-uwp-camera.podspec (hunk 1 above)
Comment (elmouzak) [reply windows-camera/react-native-windows-uwp-camera/react-native-windows-uwp-camera.podspec:19]: This is an auto generated file for iOS. I can probably delete it

**Review by dipidoo — APPROVED** [review]
lgtm

File: windows-camera/react-native-windows-uwp-camera/react-native-windows-uwp-camera.podspec (hunk 1 above)
Comment (elmouzak) [reply windows-camera/react-native-windows-uwp-camera/react-native-windows-uwp-camera.podspec]: OK, file has been deleted
